src/crimson/mon/MonClient.h

```diff
@@ -62,6 +64,13 @@ class Client : public crimson::net::Dispatcher,
 
   crimson::net::Messenger& msgr;
 
+  LogClient *log_client;
+  bool more_log_pending = false;
+  utime_t last_send_log;
+
+  seastar::future<> send_log(bool flush = false);
```

Comment:
These optional arguments, esp when they have a non-descriptive type ('bool'), are hard to read: one has to search for the function declaration to understand what is the diff between send_log() and send_log(true).
I'd suggest adding (in mon_client.h):
enum class log_flushing_t {
  no_flush,
  flush
};

Then making the function accept a non-optional parameter.
I see only 4 calls to send_log(), so it seems worth the effort